app/views/doorkeeper/authorized_applications/index.html.erb

```diff
@@ -6,53 +6,55 @@
     <h1>
       My Applications
     </h1>
-    <h2>
-      Developer applications
-      <div class="pull-right">
-        <%= link_to 'New Application', new_oauth_application_path, class: 'btn btn-success' %>
-      </div>
-    </h2>
+    <% if !@applications.empty? %>
+      <h2>
+        Developer applications
+        <div class="pull-right">
+          <%= link_to 'New Application', new_oauth_application_path, class: 'btn btn-success' %>
```

Comment:
We need to show the "New Application" button at all times, otherwise a new user will have zero applications and without the button won't be able to create a new application.